tck/src/main/java/org/apache/jdo/tck/pc/fieldtypes/FieldsOfBoolean.java

```diff
@@ -36,7 +36,7 @@ public class FieldsOfBoolean {
   private transient Boolean Boolean12;
   private transient Boolean Boolean13;
   private transient Boolean Boolean14;
-  private final Boolean Boolean15 = new Boolean(false);
+  private final Boolean Boolean15 = Boolean.valueOf(false);
```

Comment:
It's not important, but instead of Boolean.valueOf(false) you could just use Boolean.FALSE.

Reply:
That makes sense. Fixed.

Reply:
With the added benefit of making the TCK run 0.04% faster!